### DISCUSSION TITLE
Python3 error while compiling on cluster

### DISCUSSION THREAD

**Discussion by favinom** [idaholab/moose#17300] [Q&A Getting Started] [answered]
Hi all,

I am compiling moose on my cluster, it worked to quite some versions ago.
Now, when I go in test and make, I get

/usr/bin/env: python3: No such file or directory
[]/moose/framework/moose.mk:15: *** Build failed.  Stop.

Does it mean that I need python3?
We don't have it on the cluster and it always worked.

Moreover, if I look at the line 15 of moose.mk, there is nothing concerning pyhton, but it is the part concerning CONDA.

Many thanks
Marco

**Comment by ngrilli** [comment, marked as answer]
Dear @favinom 

Are you using the installation procedure with conda or manual installation gcc/mpich?
I believe if you use the manual installation, python3 on the cluster is not necessary for compiling moose.
There will be a python3 inside the miniconda installation.

Trust this helps.

Best Regards,
Nicolò Grilli
National University of Singapore

**Comment by favinom** [comment]
I am doing a manual installation.

Actually python3 is now on the cluster and I managed to compile moose.
It still gives problems but they will be object of another thread.

Grazie
